.circleci/config.yml

```diff
@@ -124,6 +124,7 @@ jobs:
       - run: make -C api test-service
       - run: make -C api test-v2
       - run: make -C api test-dao
+      - run: make -C api tests
```

Comment:
If we're adding this can we get rid of the other ones that run individually?

Reply:
was going to wait for flake and then change it all to "make test"